.github/workflows/eslint.yml

```diff
@@ -0,0 +1,50 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# ESLint is a tool for identifying and reporting on patterns
+# found in ECMAScript/JavaScript code.
+# More details at https://github.com/eslint/eslint
+# and https://eslint.org
+
+name: ESLint
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: ["main"]
+  schedule:
```

Comment:
I think this schedule task could be omitted

Reply:
you are right, it just schedules a recurrent run. Those are the details: https://docs.github.com/es/actions/using-workflows/events-that-trigger-workflows#schedule